config/default/manager/manager.yaml

```diff
@@ -67,11 +67,11 @@ spec:
                 key: END_POOL_RANGE
         resources:
           limits:
-            cpu: 100m
-            memory: 30Mi
+            cpu: 800m
```

Comment:
do we need to raise CPU? the problem is in memory

Reply:
yes I also saw a CPU issue when we run the initMap function on a large cluster